ui.apps/src/main/content/jcr_root/apps/asset-share-commons/clientlibs/clientlib-site/js/search/quick-actions.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Asset Share Commons
+ *
+ * Copyright [2017]  Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*global jQuery: false, AssetShare: false */
+jQuery((function($,ns) {
```

Comment:
Same as above, if we can always trigger the "asc.page.load" on load, and then let the handlers NOOP if there are no add/remove asset-share-ids on the page for add/remove.

Reply:
Added code to trigger "asset-share-commons.page.load" on document.ready if there is add/remove asset-share-ids on the page